### ISSUE TITLE
New Page: large list

### ISSUE THREAD

**Issue by mrlacey** [Microsoft/WindowsTemplateStudio#520]
Add a page including a large list of data

**Comment by mrlacey** [comment]
Should also handle things like:
- virtualization
- reordering
- search
- pull to refresh

more...?

**Comment by mvegaca** [comment]
I've two more ideas:
- Data filtering
- Pagination

**Comment by mrlacey** [comment]
+ grouping?

**Comment by mvegaca** [comment]
Yes

**Comment by corwyndev** [comment]
you mean server side virtualization through REST/JSON APIs?

**Comment by mrlacey** [comment]
> you mean server side virtualization through REST/JSON APIs?

No, virtualization of data loading within controls.

**Comment by mrlacey** [comment]
also
- swipe to dismiss/delete
- adding items?

**Comment by mrlacey** [comment]
Should this page be just a list or a version of the master/detail?  
If just a list, should anything be done when an item is selected? If so, what?

**Comment by ralarcon** [comment]
Not sure... looks like it would perfectly fit as a master/detail, but at the same time, it makes sense for me to have a page just with a list of items (with a more or less complex item template) that when selected/clicked you make an action (navigate to other page or any other thing), i.e I can imagine a list of songs or lines of a quote/bill...

**Comment by mrlacey** [comment]
Yeah, we already have a master/detail. It would, therefore, make sense have this as just a list.  
I think, maybe put this with nothing behind an item being selected and leave that as a TODO.
